pymeasure/adapters/protocol.py

```diff
@@ -129,7 +129,7 @@ def _read(self, **kwargs):
         """Return an already present or freshly fetched read buffer as a string."""
         return self._read_bytes(-1).decode("utf-8")
 
-    def _read_bytes(self, count, **kwargs):
+    def _read_bytes(self, count, break_on_termchar=False, **kwargs):
         """Read `count` number of bytes from the buffer.
```

Comment:
Should we call out in the docstring that `break_on_termchar` is not honored? (and maybe warn/raise if it's not False)?

Reply:
The difficulty is, how to implement it, as we do not have termchars in the ProtocolAdapter.
If we raise a NotImplemented Warning at `if break_on_termchar:`, a test could verify, that the correct form is called.

Reply:
I raise a warning now. If you want to test, that the correct parameter is called, you can verify, that the warning has been raised.